ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/CreateTableCommand.java

```diff
@@ -60,7 +60,12 @@ public CreateTableCommand(
   @Override
   public DdlCommandResult run(final MetaStore metaStore) {
     if (registerTopicCommand != null) {
-      registerTopicCommand.run(metaStore);
+      try {
+        registerTopicCommand.run(metaStore);
+      } catch (KsqlException e) {
+        throw new KsqlException(String.format("Cannot create table '%s': %s",
```

Comment:
As above, add `e` to the thrown exception.

Reply:
Done